Bottom: 861feef1c90f0b7d765d4a72e62c53c740c5bb46
Top:    f7b110c9ac20f14f56ad6ab17af40ca61bc03621
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 21:14:33 -0700

fixup-i-dont-know-where


---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index b3c8222d0f6..2536a541036 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -552,7 +552,6 @@ class FileHandleTestCase(lldbtest.TestBase):
 
     @add_test_categories(['pyapi'])
     @skipIf(py_version=['<', (3,)])
-    @skipIf(True) # fixme multiple problems with this
     def test_string_out(self):
         f = io.StringIO()
         status = self.debugger.SetOutputFile(f)
